Formula/ladspa-sdk.rb

```diff
@@ -0,0 +1,46 @@
+class LadspaSdk < Formula
+  desc "Linux Audio Developer's Simple Plugin"
+  homepage "https://ladspa.org"
+  url "https://www.ladspa.org/download/ladspa_sdk_1.15.tgz"
+  sha256 "4229959b09d20c88c8c86f4aa76427843011705df22d9c28b38359fd1829fded"
+  license "LGPL-2.1-or-later"
+
+  depends_on :linux
+
+  def install
+    cd "src" do
+      inreplace "Makefile" do |s|
```

Comment:
Can/has this been submitted upstream?

Reply:
Hmm actually we might be able to pass these through `make`